### PR TITLE
[edu] adding accessory

### PR DESCRIPTION
**Pull request by angieyth**
after creating the bowtie

Screenshot 2023-10-20 at 2 06 38 PM

after wearing the bowtie. 
the bowtie disappear from my wallet but be owned by the aptogotchi

### CODE REVIEW

**Review by darren-wangg — APPROVED** [review]

**Comment by darren-wangg** [comment]
In the future, do we want the Aptogotchi image to update and show it wearing the bowtie? Or will these always be 2 separate tokens, just overlaid on each other in the web app.

**Comment by angieyth** [comment]
when we own the accessory but not wear it, it should show up in the wallet with the bowtie image. 
when aptogotchi wear the accessory, theoratically we should show a aptogotchi wearing the bowtie in the wallet. but I don't think this overlay is supported yet

**Comment by angieyth** [comment]
in the app I think overlay is good enough for us

File: move/sources/aptogotchi.move [hunk 1]
```diff
@@ -307,4 +396,36 @@ module aptogotchi::main {
...
+        let accessory_address = get_accessory_address(creator_address, utf8(ACCESSORY_CATEGORY_BOWTIE));
+        let aptogotchi_address = get_aptogotchi_address(creator_address);
+
+        let accessory_obj = object::address_to_object<Accessory>(accessory_address);
+        assert!(object::is_owner(accessory_obj, address_of(creator)), 2);
+
+        wear_accessory(creator, utf8(ACCESSORY_CATEGORY_BOWTIE));
+        assert!(object::is_owner(accessory_obj, aptogotchi_address), 3);
+    }
 }
```
Comment (0x-j) [inline move/sources/aptogotchi.move:432]: new line

**Review by 0x-j — APPROVED** [review]

**Pull request merged**